tensorflow/python/ops/array_ops.py

```diff
@@ -4622,12 +4622,36 @@ def squeeze_v2(input, axis=None, name=None):
   # 't' is a tensor of shape [1, 2, 1, 3, 1, 1]
   tf.shape(tf.squeeze(t, [2, 4]))  # [1, 2, 3, 1]
   ```
-
+  
   Unlike the older op `tf.compat.v1.squeeze`, this op does not accept a
   deprecated `squeeze_dims` argument.
 
   Note: if `input` is a `tf.RaggedTensor`, then this operation takes `O(N)`
   time, where `N` is the number of elements in the squeezed dimensions.
+  
+  Note: If squeeze is performed on dimensions of unknown sizes, then the
+  returned   Tensor will be of unknown shape. In Keras the first (batch)
```

Comment:
Please avoid mentioning Keras in low level TF API documentation. I think you can mention that in general the first dimension of Tensors using in ML training workloads is an unknown batch dimension.

Reply:
Done.